apps/onboarding/content1.markup

```diff
@@ -3,8 +3,8 @@
 
     <Container padding="insets(8, 0)" layout="vflow(2)">
         <Label text="Click on the top left of the screen to bring up the application menu." anchor="center"/>
-        <Label text="There is also a terminal with familiar unix utilities" anchor="center"/>
-        <Label text="like ls, yes, grep, cd, which, ..." anchor="center"/>
+        <Label text="Checkout the terminal with familiar unix utilities" anchor="center"/>
```

Comment:
`Checkout` should be written as `Check out`. I would suggest using `Try out` instead.